lib/ui/screens/liked_photos_screen.dart

```diff
@@ -0,0 +1,410 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:http/http.dart' as http;
+import 'package:mime/mime.dart';
+import 'package:path_provider/path_provider.dart';
+import 'package:photo_view/photo_view.dart';
+import 'package:photo_view/photo_view_gallery.dart';
+import 'package:reaxit/api/exceptions.dart';
+import 'package:reaxit/blocs.dart';
+import 'package:reaxit/blocs/liked_photos_cubit.dart';
+import 'package:reaxit/models.dart';
+import 'package:reaxit/ui/theme.dart';
+import 'package:reaxit/ui/widgets.dart';
+import 'package:reaxit/ui/widgets/photo_tile.dart';
+import 'package:share_plus/share_plus.dart';
+import 'package:gallery_saver/gallery_saver.dart';
+
+/// Screen that loads and shows the Album with `slug`.
+class LikedPhotosScreen extends StatelessWidget {
+  const LikedPhotosScreen();
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: ThaliaAppBar(
+        title: const Text('LIKED PHOTOS'),
+      ),
+      body: RefreshIndicator(
+        onRefresh: () => BlocProvider.of<LikedPhotosCubit>(context).load(),
```

Comment:
@JeeVee11 I don't see where this cubit would be provided now. You can surround the scaffold with a blocprovider that creates a cubit, such that the blocbuilder etc can then access it. That should solve the providernotfound stuff

Reply:
It could also be provided in main.dart lazily, but since this won't be accessed frequently doing it here (so losing its state when you leave the screen) is cleaner